install_config/configuring_authentication.adoc

```diff
@@ -1053,6 +1053,43 @@ a value is used as the display name. Optional, case-insensitive.
 identity determined from the headers specified in `headers`. The first header containing
 a value is used as the preferred user name when provisioning. Optional, case-insensitive.
 
+[discrete]
+[[windows-sspi-using-request-header]]
+==== SSPI connection support on Windows
+
+ifdef::openshift-enterprise[]
+
+[IMPORTANT]
+====
+Using SSPI connection support on Windows is a Technology Preview feature.
```

Comment:
I would say Microsoft Windows to ensure proper branding, but otherwise looks good! :+1: